### PR TITLE
Prepare spring cloud azure 6.0.0-beta.3 release

### PR DESCRIPTION
**Pull request by moarychan**
As the title to prepare Spring Cloud Azure 6.0.0-beta.3 release, which is based on Spring Boot 3 RC1 and Spring Cloud 2022 RC1.

### CODE REVIEW

**Comment by moarychan** [comment]
We will ignore the tests job failed, more detail from https://github.com/Azure/azure-sdk-for-java/pull/31889#pullrequestreview-1166567750

File: sdk/spring/spring-cloud-azure-actuator-autoconfigure/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 6.0.0-beta.3 (Unreleased)
+## 6.0.0-beta.3 (2022-11-04)
```
Comment (chenrujun) [inline sdk/spring/spring-cloud-azure-actuator-autoconfigure/CHANGELOG.md:3]: How about adding the information about spring-boot version and spring-cloud version.

File: sdk/spring/spring-cloud-azure-actuator/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 6.0.0-beta.3 (Unreleased)
+## 6.0.0-beta.3 (2022-11-04)
```
Comment (chenrujun) [inline sdk/spring/spring-cloud-azure-actuator/CHANGELOG.md:3]: Adding content like `## 4.3.0 (2022-06-29) Please refer to ...`

Comment (chenrujun) [reply sdk/spring/spring-cloud-azure-actuator/CHANGELOG.md:3]: Same to all other places.

File: sdk/spring/spring-cloud-azure-actuator-autoconfigure/CHANGELOG.md (hunk 1 above)
Comment (moarychan) [reply sdk/spring/spring-cloud-azure-actuator-autoconfigure/CHANGELOG.md:3]: This info has been added here https://github.com/moarychan/azure-sdk-for-java/blob/release/spring-cloud-azure-6.0.0-beta.3/sdk/spring/CHANGELOG.md#600-beta3-2022-11-04

File: sdk/spring/spring-cloud-azure-actuator-autoconfigure/CHANGELOG.md [hunk 3]
```diff
@@ -12,6 +14,8 @@
 
 ## 6.0.0-beta.2 (2022-09-30)
 
+Please refer to [CHANGELOG.md](https://github.com/Azure/azure-sdk-for-java/blob/4f0af481662eebd1703a4082eea9a291b76c1608/sdk/spring/CHANGELOG.md#600-beta2-2022-09-30) for more details.
```
Comment (chenrujun) [inline sdk/spring/spring-cloud-azure-actuator-autoconfigure/CHANGELOG.md:17]: 1. We should always update the commit id (4f0af481662eebd1703a4082eea9a291b76c1608) to the latest one. Then the link will display all release versions. 
2. We should add prefix of CHANGELOG to spring/CHANGELOG to make it less confuses without open the link

Of course, it's not caused by this PR. Could you please create an issue? And add a task item in that issue: Update release wiki.

File: sdk/spring/spring-cloud-azure-service/CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 6.0.0-beta.3 (Unreleased)
+Please refer to [CHANGELOG.md](https://github.com/Azure/azure-sdk-for-java/blob/63d3802ccb6e36062410f28ff90cae60de6ea3a4/sdk/spring/CHANGELOG.md#600-beta3-2022-11-04) for more details.
```
Comment (chenrujun) [inline sdk/spring/spring-cloud-azure-service/CHANGELOG.md:3]: The version should not be deleted.

**Review by chenrujun — CHANGES_REQUESTED** [review]

File: sdk/spring/spring-cloud-azure-actuator-autoconfigure/CHANGELOG.md (hunk 3 above)
Comment (moarychan) [reply sdk/spring/spring-cloud-azure-actuator-autoconfigure/CHANGELOG.md:17]: Opened https://github.com/Azure/azure-sdk-for-java/issues/31930

**Review by chenrujun — APPROVED** [review]
LGTM.

**Pull request merged**